benchmarks/common.py

```diff
@@ -668,6 +736,7 @@ def run_one_model(
         tolerance, cos_similarity = self.get_tolerance_and_cosine_flag(
             is_training, current_device, name
         )
+        static_inputs = (example_inputs[0][0], *example_inputs[1:])
```

Comment:
Worried that it will cause other models to break? For example, hugggingface.py models have example_inputs as dict.

Reply:
ah. yea, this is wrong. i just did it as a hack.  forgot to fix.  hmm, i will need to do something more to integrate this properly.

i also need to fix how 'warmup' works for ds, so maybe the thing to do is just short-circuit all of this in the --dynamic-shapes case, jump straight to my `speedup-experiment-ds`, and do the warmup there.